src/slackHandlers/appMention.js

```diff
@@ -17,6 +17,9 @@ const { helpRequestDuplicateBlocks } = require("../messages");
 const { lookupUsersName } = require("./utils/lookupUser");
 const reportChannelId = config.get("slack.report_channel_id");
 
+const feedback =
```

Comment:
extracted to variable so this is kept in sync with the analytics code that looks for this text